### PR TITLE
Update samples yaml

### PR DESCRIPTION
**Pull request by ryantd**
## Why are these changes needed?

1. Update configmap
    1. Rename the script to avoid a "`ray has no attribute init`" issue [1]
    2. Fix `services.get_node_ip_address`
    3. Change `ray_nodes` dir to avoid a `Permission error`
2. Update yaml
    1. Bump rayVersion to `1.8.0`
    2. Bump ray image version to `1.8.0`
    3. Change command: `sleep infinity`
    4. Add a comment on `init-myservice`'s command

[1]:
```
trying to connect to Ray!
Traceback (most recent call last):
  File "/opt/code.py", line 1, in <module>
    import ray
  File "/home/ray/anaconda3/lib/python3.7/site-packages/ray/__init__.py", line 91, in <module>
    import ray._raylet  # noqa: E402
  File "python/ray/_raylet.pyx", line 118, in init ray._raylet
  File "/home/ray/anaconda3/lib/python3.7/site-packages/ray/_private/runtime_env/validation.py", line 11, in <module>
    from ray._private.runtime_env.plugin import RuntimeEnvPlugin
  File "/home/ray/anaconda3/lib/python3.7/site-packages/ray/_private/runtime_env/plugin.py", line 3, in <module>
    from ray.util.annotations import DeveloperAPI
  File "/home/ray/anaconda3/lib/python3.7/site-packages/ray/util/__init__.py", line 14, in <module>
    from ray.util import rpdb as pdb
  File "/home/ray/anaconda3/lib/python3.7/site-packages/ray/util/rpdb.py", line 16, in <module>
    from pdb import Pdb
  File "/home/ray/anaconda3/lib/python3.7/pdb.py", line 76, in <module>
    import code
  File "/opt/..2021_11_19_09_25_09.256271217/code.py", line 5, in <module>
    ray.init(address="auto", _redis_password=redis_pass)
AttributeError: module 'ray' has no attribute 'init'
```
## Checks

- [x] I've made sure the tests are passing. 
- Testing Strategy
   - [ ] Unit tests
   - [x] Manual tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: ray-operator/config/samples/ray-cluster.mini.yaml [hunk 1]
```diff
@@ -64,6 +64,15 @@ spec:
             name: dashboard
           - containerPort: 10001
             name: client
-
-
-
+          volumeMounts:
```
Comment (Jeffwan) [inline ray-operator/config/samples/ray-cluster.mini.yaml:67]: Let's keep the `mini` one clean and easy to launch the cluster without additional changes?

**Comment by Jeffwan** [comment]
Thanks for the contribution, the change looks good to me because above minor comment

File: ray-operator/config/samples/ray-cluster.mini.yaml (hunk 1 above)
Comment (ryantd) [reply ray-operator/config/samples/ray-cluster.mini.yaml]: Sure ~~

**Comment by ryantd** [comment]
@Jeffwan PTAL

**Review by Jeffwan — APPROVED** [review]

**Pull request merged**